### PR TITLE
Plugins: Replace Old Analyzer Naming

### PR DESCRIPTION
**Pull request by ax3l**
Replace left-over "analy[s|z]er" speach we used before calling them *plugins*.

Done via one-liner script and commited as generic author:
```bash
grep -iR "analyser" src/ | awk -F: '{print $1}' | \
    sort -u | xargs -n1 -P1 -I{} sed -i 's/analyser/plugin/g' {}
grep -iR "analyzer" src/ | awk -F: '{print $1}' | \
    sort -u | xargs -n1 -P1 -I{} sed -i 's/analyzer/plugin/g' {}

GIT_AUTHOR_NAME="Tools" GIT_AUTHOR_EMAIL="picongpu@hzdr.de" \
    git commit -S
```

ignored:
 - src/tools/bin/radiationSyntheticDetector

### CODE REVIEW

File: src/tools/bin/radiationSyntheticDetector [hunk 1]
```diff
@@ -52,7 +52,7 @@ parser.add_argument('--dataExtend', '-d',
                             [default=0.5 3.5 0.0 360.0]''')
 
 # extend of the data to analyze
-parser.add_argument('--analyzerExtend', '-a',
+parser.add_argument('--pluginExtend', '-a',
```
Comment (PrometheusPi) [inline src/tools/bin/radiationSyntheticDetector]: REQUIRED:
please revert this - analyzer is the data analyzer in this tool - not a plugin

File: src/tools/bin/radiationSyntheticDetector [hunk 2]
```diff
@@ -83,11 +83,11 @@ for myfile in args.path2Data:
...
-    # set extent analyzer:
-    omega_min_analy = args.analyzerExtend[0]
-    omega_max_analy = args.analyzerExtend[1]
-    theta_min_analy = args.analyzerExtend[2]
-    theta_max_analy = args.analyzerExtend[3]
+    # set extent plugin:
+    omega_min_analy = args.pluginExtend[0]
+    omega_max_analy = args.pluginExtend[1]
+    theta_min_analy = args.pluginExtend[2]
+    theta_max_analy = args.pluginExtend[3]
```
Comment (PrometheusPi) [inline src/tools/bin/radiationSyntheticDetector]: REQUIRED:
same here

Comment (ax3l) [inline src/tools/bin/radiationSyntheticDetector]: sure!

but is "analyzer" a great name here either? maybe you want to express something like "selection" or "area of interest" or "extent" here?

Comment (PrometheusPi) [inline src/tools/bin/radiationSyntheticDetector]: @ax3l I agree - selection or ROI is a better name for this variable
 I will fix it in another pull request.

Comment (PrometheusPi) [inline src/tools/bin/radiationSyntheticDetector]: Sorry - I was in the wrong code - here it is a **statistical analysis** this the name is correct (ROI, etc. would be confusing). 
Just leave it as it is.

Comment (ax3l) [inline src/tools/bin/radiationSyntheticDetector]: ok

Comment (ax3l) [inline src/tools/bin/radiationSyntheticDetector]: > Just leave it as it is.

oh, you mean *revert those lines*?

Comment (PrometheusPi) [inline src/tools/bin/radiationSyntheticDetector]: exactly

**Pull request merged**